server/src/storage/data_transfer.rs

```diff
@@ -261,34 +271,28 @@ mod tests {
 
     #[test]
     fn test_folder_path_is_not_compressed_file() {
-        let path = ObjectStorePath::from("4330327753845164038/compressed");
-        assert!(DataTransfer::path_is_compressed_file(path).is_none());
-    }
-
-    #[test]
-    fn test_non_key_folder_is_not_compressed_file() {
-        let path = ObjectStorePath::from("test/compressed/test.parquet");
+        let path = ObjectStorePath::from(format!("compressed/{}", TABLE_NAME));
         assert!(DataTransfer::path_is_compressed_file(path).is_none());
     }
 
     #[test]
-    fn test_key_folder_without_compressed_folder_is_not_compressed_file() {
-        let path = ObjectStorePath::from("4330327753845164038/test/test.parquet");
+    fn test_table_folder_without_compressed_folder_is_not_compressed_file() {
+        let path = ObjectStorePath::from(format!("test/{}/test.parquet", TABLE_NAME));
         assert!(DataTransfer::path_is_compressed_file(path).is_none());
     }
 
     #[test]
     fn test_non_parquet_file_is_not_compressed_file() {
-        let path = ObjectStorePath::from("4330327753845164038/compressed/test.txt");
+        let path = ObjectStorePath::from("compressed/table/test.txt");
```

Comment:
Maybe also use the TABLE_NAME constant here.

Reply:
Fixed.